src/app/dtos/service.report.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsArray, IsString } from 'class-validator';
+
+/**
+ * Please note: this structure is expected to be used across all services
+ * which report statistics
+ */
```

Comment:
At some point will we move this DTO into `protocol-common`?

Reply:
yes.   we have some problems with protocol-common which we need to resolve :/